### ISSUE TITLE
A new algorithm to generate more cool names

### ISSUE THREAD

**Issue by mohitkyadav** [mohitkyadav/cool-name-api#3]
Currently this app replaces some characters in a given string.

**Comment by mufti1** [comment]
can describe this app? maybe i am can help

**Comment by mohitkyadav** [comment]
yes sure, it's a simple express based javascript web app, it simply replaces some characters from a given string and returns with a JSON response with 4 strings.
Take a look [here](https://github.com/mohitkyadav/cool-name-api/blob/master/services/coolifyService.js#L92) for more info.

**Comment by PROTechThor** [comment]
Will try this out.
